lib/model/autocomplete.dart

```diff
@@ -289,6 +289,8 @@ class MentionAutocompleteQuery {
     // TODO(#236) test email too, not just name
     // TODO(#237) test with diacritics stripped, where appropriate
 
+    if (!user.isActive) return false;
+
     final List<String> nameWords = cache.nameWordsForUser(user);
 
     int nameWordsIndex = 0;
```

Comment:
This function will get confusing if we start adding more and more features to it and include all the details in the same function.

Let's have a prep NFC commit that factors out a helper method `_testName`, with the same signature as `testUser`. Then the main commit can add the `user.isActive` check to `testUser`.